bin/qds.py

```diff
@@ -550,7 +562,14 @@ def main():
                      api_url=options.api_url,
                      version=options.api_version,
                      poll_interval=options.poll_interval,
-                     skip_ssl_cert_check=options.skip_ssl_cert_check)
+                     skip_ssl_cert_check=options.skip_ssl_cert_check,
+                     cloud=options.cloud)
+
+    # If cloud provider is not set by user at system level or through command line ,
+    # then fetch cloud from api url request
+    if options.cloud is None:
+        options.cloud = Cloud.get_cloud()
```

Comment:
We can set default cloud provider to `aws`. I think there is no need to make extra api call just to fetch the cloud provider.